std_msgs/Int64 justDetected
std_msgs/Int64 closeBall
std_msgs/Int64 radius
std_msgs/Int64 circCenter
